src/qt/test/test_main.cpp

```diff
@@ -14,6 +14,7 @@
 #include <qt/test/uritests.h>
 #include <qt/test/compattests.h>
 #include <test/setup_common.h>
+#include <util/time.h>
```

Comment:
No longer needed after rebase

Reply:
Good catch!

Re-ran my analysis script and it turns out that `util/time.h` and `chainparams.h` are no longer needed.